### PR TITLE
Show diff when apply usergroups with dryrun

### PR DESCRIPTION
**Pull request by takaishi**

### CODE REVIEW

File: lib/arisaid/usergroups.rb [hunk 1]
```diff
@@ -47,26 +47,65 @@ def apply
...
+
+        if users_changed?(src, dst)
+          add = src['users'].flatten.sort  - dst['users'].flatten.sort
+          delete = dst['users'].flatten.sort - src['users'].flatten.sort
+          add.each do |u|
+            puts "  + user #{u}"
+          end
+          delete.each do |u|
+            puts "  - user #{u}"
+          end
```
Comment (linyows) [inline lib/arisaid/usergroups.rb]: I always seem to see diff. Correct?

File: lib/arisaid/usergroups.rb [hunk 2]
```diff
@@ -47,26 +47,65 @@ def apply
...
+        if changed?(src, dst)
+          puts "update usergroup: #{src['name']}"
+          update(src)
+        end
+
+        if users_changed?(src, dst)
+          add = src['users'].flatten.sort  - dst['users'].flatten.sort
+          delete = dst['users'].flatten.sort - src['users'].flatten.sort
+          add.each do |u|
+            puts "  + user #{u}"
```
Comment (linyows) [inline lib/arisaid/usergroups.rb]: fmfm--

**Comment by takaishi** [comment]
I agree to separate display and apply, I splitted to display diff and call slack api. And I changed to run display diff only dry-run.

File: lib/arisaid/usergroups.rb (hunk 1 above)
Comment (takaishi) [reply lib/arisaid/usergroups.rb]: This should to display only dry-run, I fixed.

**Pull request merged**